### DISCUSSION TITLE
Getting below error suddenly. I was able to see the data before.

### DISCUSSION THREAD

**Discussion by sayaligo** [grafana/grafana-infinity-datasource#995] [Q&A]
Object
status:500
statusText:""
data:Object
message:"An error occurred within the plugin"
messageId:"plugin.downstreamError"
statusCode:500
traceID:""
config:Object
url:"api/ds/query?ds_type=yesoreyeram-infinity-datasource&requestId=Q114"
method:"POST"
data:Object
requestId:"Q114"
hideFromInspector:false
headers:Object
retry:0
traceId:undefined
message:"An error occurred within the plugin"

**Comment by rmorshea-atom-computing** [comment]
I'm experiencing this as well when using an Azure-managed Grafana instance.

**Reply by rmorshea-atom-computing** [reply]
Potentially related to: https://github.com/grafana/grafana-infinity-datasource/discussions/1039

**Reply by rmorshea-atom-computing** [reply]
Also attempting to get support in [this Slack thread](https://grafana.slack.com/archives/C0Y4TLW74/p1730344208557699).
